### PR TITLE
MAP-524 Initial setup of the API

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/config/WebClientConfiguration.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+@Configuration
+class WebClientConfiguration(
+  @Value("\${api.base.url.oauth}") private val authBaseUri: String,
+) {
+
+  @Bean
+  fun webClient(builder: WebClient.Builder): WebClient {
+    return builder
+      .baseUrl(authBaseUri)
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/config/WebClientConfiguration.kt:30]: worth adding timeout to this too

File: src/main/resources/application-localstack.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+hmpps.sqs:
+  reactiveApi: true
```
Comment (petergphillips) [inline src/main/resources/application-localstack.yml:2]: not sure if you mean this to use the reactive api

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/resources/application-localstack.yml (hunk 2 above)
Comment (Mjwillis) [reply src/main/resources/application-localstack.yml:2]: no definitely not!

**Pull request merged**